x/btcstaking/keeper/keeper.go

```diff
@@ -70,10 +70,8 @@ func (k Keeper) BeginBlocker(ctx context.Context) error {
 	}
 	wValue := k.btccKeeper.GetParams(ctx).CheckpointFinalizationTimeout
 
-	// prepare for recording finality providers with positive voting power
-	activeFps := []*types.FinalityProviderWithMeta{}
 	// prepare for recording finality providers and their BTC delegations
-	// for rewards
+	// for voting power and rewards
 	rdc := types.NewRewardDistCache()
```

Comment:
```suggestion
	rdc := types.NewVotingPowerDistCache()
```
I just feel like this is essentially what this cache stores. Or we can rename it something similar.

Reply:
Good point. I was also wondering whether to update the name since the struct is also used for voting power table now. Renamed `RewardDistCache` to `VotingPowerDistCache` :+1: